sdk/synapse/Azure.Analytics.Synapse.Spark/src/Generated/Models/SparkStatementCancellationResult.cs

```diff
@@ -16,12 +16,13 @@ internal SparkStatementCancellationResult()
         }
 
         /// <summary> Initializes a new instance of SparkStatementCancellationResult. </summary>
-        /// <param name="msg"> . </param>
-        internal SparkStatementCancellationResult(string msg)
+        /// <param name="message"> The msg property from the Livy API. The value is always &quot;canceled&quot;. </param>
```

Comment:
Is `msg` a name from the API or an abbreviation?  Would it be worth marking it bold or with quotes to indicate it's not an abbreviation?

Reply:
Yes, this is generated code from the swagger (https://github.com/Azure/azure-rest-api-specs/commit/50b8835320ea4e54d64b57c9394632b504293183#diff-6a8d6de11eb6b2bdf84d780437921ff738272e63b3daceec2a47d1fa1ffae925).

Reply:
I also midread the diff, we're going from msg to Message (correctly).

Reply:
Should the comment be updated to reflect this?

Reply:
I'm not sure here. We're taking 'msg' from the Livy API and giving it the name message. It's just a shortened version.

The comment here is from the service team.

Reply:
It's a super nitpicky comment - but would it make sense to put `msg` in quotes to indicate it's not a typo?